packages/explorer-ui/pages/tx/[kappa].tsx

```diff
@@ -37,12 +37,14 @@ export const BridgeTransaction = ({ queryResult }) => {
   const transaction = queryResult.bridgeTransactions[0]
   const { pending, fromInfo, toInfo } = transaction
 
-  const getTimeDifference = (start, end) => {
-    const diff = end - start
-    if (0 >= diff) {
-      return '1'
+  // Get time taken to complete tx w/appropriate units.
```

Comment:
```suggestion
  // Get time taken to complete tx w/ appropriate units.
```

Reply:
we can address this in a future PR, ill open a task for you

Reply:
fixed!

Reply:
LGTM